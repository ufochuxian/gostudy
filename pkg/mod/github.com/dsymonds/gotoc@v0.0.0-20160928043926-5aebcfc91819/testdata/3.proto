// Default values.

syntax = "proto2";

message A {
  optional int32 int = 1 [default=7];
  optional bool boolean = 2 [default=true];
  optional string str = 3 [default="boo"];
}
