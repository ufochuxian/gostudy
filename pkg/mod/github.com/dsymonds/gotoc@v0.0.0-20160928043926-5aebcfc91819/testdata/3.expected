file: <
  name: "3.proto"
  message_type: <
    name: "A"
    field: <
      name: "int"
      number: 1
      label: LABEL_OPTIONAL
      type: TYPE_INT32
      default_value: "7"
    >
    field: <
      name: "boolean"
      number: 2
      label: LABEL_OPTIONAL
      type: TYPE_BOOL
      default_value: "true"
    >
    field: <
      name: "str"
      number: 3
      label: LABEL_OPTIONAL
      type: TYPE_STRING
      default_value: "boo"
    >
  >
>
